### ISSUE TITLE
Is it possible for you to upload the models to google drive or send them to gmail?

### ISSUE THREAD

**Issue by Kyanite123** [WangWenhao0716/Adapted-Center-and-Scale-Prediction#3]

**Comment by WangWenhao0716** [comment]
Yes, I provide three models: one more with pre-trained on EuroCity Person dataset. 
Google Drive: https://drive.google.com/drive/folders/1MS2Mh_TAdU4rJaP1Z-NiXojVQm_VYS__?usp=sharing

**Comment by Kyanite123** [comment]
It works wonderfully! Thanks!!!!

**Comment by And1210** [comment]
Could you also add the ResNet-101 model to this drive? I cannot download it as I can't get a Baidu account

**Comment by WangWenhao0716** [comment]
I think the above models are the ResNet-101 models if there is nothing wrong with my memory.

**Comment by And1210** [comment]
Well i downloaded those models but I can't seem to run any of the scripts as they all expect './models/ResNet101v2+SN(8,32).pth' which isn't included in the drive there.

**Comment by WangWenhao0716** [comment]
OK, I know. It is available at https://drive.google.com/file/d/1FGjlrkHnLOVn2COmOuEXw2O8FOpSlJxw/view?usp=sharing

**Comment by And1210** [comment]
Thank you so much! I really appreciate it
